mbuild/formats/hoomdxml.py

```diff
@@ -88,157 +71,239 @@ def write_hoomdxml(structure, filename, box, ref_distance=1.0, ref_mass=1.0,
                       k1, k2, k3, k4 : force coefficients (units of energy)
     Dihedral : system dihedrals
     Body : rigid body to which each atom belongs
+
     """
 
     forcefield = True
     if structure[0].type == '':
         forcefield = False
-
     xyz = np.array([[atom.xx, atom.xy, atom.xz] for atom in structure.atoms])
 
-    # Center box at origin and remap coordinates into box
     box.maxs *= 10.
```

Comment:
We really should be reading the box information in from the ParmEd structure, but we can worry about that in a separate PR since that has more to do with how we're passing the box information when converting an mBuild Compound to a ParmEd Structure